src/_includes/layouts/event.html

```diff
@@ -29,9 +33,45 @@
 		<div data-layout="centered-single-column">
 			<main class="events-page">
 				<h1>{{ title }}</h1>
-				<small> {% if speakers %} {{ speakers | join: ', ' }} &middot; {% endif %} {{ date }} </small>
+				<small> {% if speakers %} {{ speakers | join: ', ' }} &middot; {% endif %} {{ date | asDateTime }} </small>
 				{{ addToCalendarList }} {{ content }}
-				<p>Join this event <a href="https://discord.com/invite/q9ZdBgP">in Discord</a></p>
+				<div>
+					<span>
+						<svg
```

Comment:
We should tack a `focusable="false"` on here – this will ensure that Internet Explorer users can't tab to this `svg` which, wildly enough, is a thing that happens.

Reply:
didn't even know that was an SVG property!

just pushed up a mixup commit